src/constants/v3/addresses.ts

```diff
@@ -6,19 +6,19 @@ import { WMATIC } from 'v3lib/entities/wmatic';
 type AddressMap = { [chainId: number]: string };
 
 export const V3_CORE_FACTORY_ADDRESSES: AddressMap = {
-  [ChainId.MATIC]: '0xd556ED8F399b29504095E606feF34e53Df9B0c14',
+  [ChainId.MATIC]: '0x8C1EB1e5325049B412B7E71337116BEF88a29b3A',
 };
 
 export const POOL_DEPLOYER_ADDRESS: AddressMap = {
-  [ChainId.MATIC]: '0x390e1F04BF44C33F491231E7865fF05E583813C5',
+  [ChainId.MATIC]: '0x218a510d4d6aEA897961ab6Deb74443521A88839',
```

Comment:
What set of addresses are these pointing to. We should be using the same ones as the quickswap info branch on algebra?